src/views/projects/Issue.svelte

```diff
@@ -115,39 +119,31 @@
       if (add.length === 0 && remove.length === 0) {
         return;
       }
-      await api.project.updateIssue(
+      await issues.updateIssue(
         projectId,
         issue.id,
         {
           type: "assign",
           add: utils.stripDidPrefix(add),
           remove: utils.stripDidPrefix(remove),
         },
-        $httpdStore.session.id,
+        $httpdStore.session,
+        api,
       );
-      issue = await api.project.getIssueById(projectId, issue.id);
+      issue = await issues.refreshIssue(projectId, issue, api);
     }
   }
 
   async function saveStatus({ detail: state }: CustomEvent<IssueState>) {
     if ($httpdStore.state === "authenticated") {
-      await api.project.updateIssue(
+      await issues.updateIssue(
         projectId,
         issue.id,
         { type: "lifecycle", state },
-        $httpdStore.session.id,
+        $httpdStore.session,
+        api,
       );
-      void router.push({
```

Comment:
Removing the push breaks issue counter updates in the header when changing the issue state. We can maybe leave the push as is, since if the request fails the user will still land on an error page.

Reply:
@rudolfs added issue #897 to implement the header counter update test